orbit/changes/25987-installed-win-updates-only

```diff
@@ -0,0 +1 @@
+When querying the 'windows_updates' Orbit table return non-installed windows updates only.
```

Comment:
Please add a bullet `*` and use past tense (see [here](https://github.com/fleetdm/fleet/blob/main/docs/Contributing/Releasing-Fleet.md?plain=1#L17), I'll update the changes docs to make this clear).

Reply:
Done in b4f731decdf38df5a233fb84cfb6dc5fdafbf740